pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -175,12 +194,13 @@ func (r *ReplicaSetReconciler) resetStatefulSetUpdateStrategy(mdb mdbv1.MongoDB)
 
 // isStatefulSetReady checks to see if the stateful set corresponding to the given MongoDB resource
 // is currently in the ready state
-func (r *ReplicaSetReconciler) isStatefulSetReady(mdb mdbv1.MongoDB) (bool, error) {
+func (r *ReplicaSetReconciler) isStatefulSetReady(mdb mdbv1.MongoDB, expectedUpdateStrategy appsv1.StatefulSetUpdateStrategyType) (bool, error) {
+	time.Sleep(time.Second * 5)
```

Comment:
we should add a comment and TODO here, maybe even an issue so this doesn't stay as a long term solution.

Reply:
commented on the code

https://jira.mongodb.org/browse/CLOUDP-62444